tests/actions/ReportTest.ts

```diff
@@ -1219,7 +1236,11 @@ describe('actions/Report', () => {
 
     it('should send DeleteComment request and remove any Reactions accordingly', async () => {
         global.fetch = TestHelper.getGlobalFetchMock();
-        jest.spyOn(EmojiUtils, 'hasAccountIDEmojiReacted').mockImplementation(() => true);
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-return
+        jest.doMock('@libs/EmojiUtils', () => ({
+            ...jest.requireActual('@libs/EmojiUtils'),
+            hasAccountIDEmojiReacted: jest.fn(() => true),
+        }));
```

Comment:
Apologies about that other comment - I think I meant to ask about THIS change, not the one above. Was this a lint thing?

Reply:
Ah, yes, this is because the lint complaint when we import 
`import * as EmojiUtils from '@libs/EmojiUtils';`

so we can't do `jest.spyOn(EmojiUtils, 'hasAccountIDEmojiReacted')` anymore. So instead, I mock the whole module using doMock (so it's mocked for this test case only), but only mock the `hasAccountIDEmojiReacted` function.

Reply:
ah okay cool, thank you for explaining!